File: NCC_API/Models/DanhMuc/ChucDanhModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Timensit_API.Models.DanhMuc
{
   
    public class ChucDanhModel
    {
        public long id_cv { get; set; } = 0;
        public string Cap { get; set; }
        public string TenCV { get; set; }
        public string MaCV { get; set; }
        public bool IsManager { get; set; }
        public bool IsTaiXe { get; set; }
    }
}

File: Timensit_Backend/TimensitApp/src/app/views/pages/Timensit/danh-muc/chucdanh/model/chucdanh.model.ts
import { BaseModel } from '../../../../../../core/_base/crud';

export class ChucDanhModel extends BaseModel {
	Id_CV: number;
	MaCV: string;
	TenCV: string;
	Cap: string;
	IsManager: boolean;
	IsTaiXe: boolean;
	clear() {
		this.Id_CV = 0;
		this.MaCV = '';
		this.TenCV = '';
		this.Cap = '';
		this.IsManager = false;
		this.IsTaiXe = false;
	}
}
